_summaries/katharopoulos2020lineartransformer_2.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: summary
+title: Summary
+paper: {{katharopoulos2020lineartransformer}}
+author:  sritank
+score:  10
+---
+
+## What is the core idea?
+A new linear transformer model is introduced that uses a kernel based self-attention method that reduces the memory footprint of the model. They also linearize the masking of the attention computation making it run in linear time.
```

Comment:
Another important idea in the paper was using their causal masked attention to express a transformer layer as a Recurrent Neural Network. Might be a good idea to add that to the core idea section.